### DISCUSSION TITLE
On MacOS add Applications folder to dmg for drag and drop install

### DISCUSSION THREAD

**Discussion by CubeOfCheese** [godotengine/godot-proposals#9498] [Platforms]
Most MacOS dmgs have an Applications folder so you can click and drag the application to the Applications folder immediately. 

Generic example: 
![maxresdefault-901303597](https://github.com/godotengine/godot-proposals/assets/16740469/e591fd43-8ba6-4523-9477-077156960676)

It would be nice to have that as an option in the godot export, if not by default.

Relevant blog post explaining how to script this: https://asmaloney.com/2013/07/howto/packaging-a-mac-os-x-application-using-a-dmg/

**Comment by bruvzg** [comment]
Adding symlink to the `Applications` should be easy. Setting background and other stuff is not hard as well (can be done via AppleScript/Automator), but it will need some dedicated UI to configure it (and probably an option to add other files to the image).
